xbuild/src/gradle/ic_launcher.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+<!--
+  <background android:drawable="@mipmap/ic_launcher_background"/>
+-->
+  <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
+  <monochrome android:drawable="@mipmap/ic_launcher_monochrome"/>
+</adaptive-icon>
```

Comment:
missing trailing new line

Reply:
Added the newline but github web ui does not seem to reflect this change. 
in my ide: ![image](https://user-images.githubusercontent.com/23728505/208566789-d04b87ae-e4ff-4cc8-b7f9-588b3bf662c0.png)

Reply:
GitHub UI only reflects when it is _not_ there:

![image](https://user-images.githubusercontent.com/2325264/209001878-d26e5092-e08b-4492-806a-6e2cc91dca70.png)